src/fpylll/contrib/simple_bkz.py

```diff
@@ -110,3 +110,59 @@ def svp_reduction(self, kappa, block_size):
             self.m.remove_last_row()
 
         return False
+
+class DBKZReduction(BKZReduction):
```

Comment:
I personally would prefer this class to have its own file, but won't insist on it.